tests/test_modeling_flax_common.py

```diff
@@ -160,12 +160,58 @@ def recursive_check(tuple_object, dict_object):
             dict_inputs = self._prepare_for_class(inputs_dict, model_class)
             check_equivalence(model, tuple_inputs, dict_inputs, {"output_hidden_states": True})
 
+    def check_outputs(self, fxo, pto, model_class, names):
```

Comment:
(nit) could we call this `fx_outputs` and `pt_outputs` instead of `fxo` and `pto` ?

Reply:
Sure, you and Sylvain told me the same thing :)